index.bs

```diff
@@ -300,6 +300,7 @@ interface Blob {
   [NewObject] ReadableStream stream();
   [NewObject] Promise<USVString> text();
   [NewObject] Promise<ArrayBuffer> arrayBuffer();
+  [NewObject] Promise<ArrayBuffer> bytes();
```

Comment:
Uh, should be `Promise<Uint8Array>` , right? Copy/paste and forgot to update?

Reply:
Whoops, yup. Fixed.